docs/connector-development/connector-builder-ui/overview.md

```diff
@@ -1,57 +1,48 @@
+3. Once the connector is ready, publish it. This makes it available in your local workspace
+4. Configure a Source based on the released connector
+5. Use the Source in a connection to sync data
 
-Follow [the tutorial](./tutorial.mdx) for an example of how this looks like in practice.
-:::
+The concept pages in this section of the docs share more details related to the following topics: [authentication](./authentication.md), [record processing](./record-processing.mdx), [pagination](./pagination.md), [incremental sync](./incremental-sync.md), [partitioning](./partitioning.md), and [error handling](./error-handling.md). 
 
-The **configured source** is configuring a connector to actually extract records. The exact fields of the configuration depend on the connector, but in most cases, it provides authentication information (username and password, API key) and information about which data to extract, for example, the start date to sync records from, a search query records have to match.
+Do not hardcode things like API keys or passwords while configuring a connector in the builder. They will be used, but not saved, during development when you provide them as Testing Values. For use in production, these should be passed in as user inputs after publishing the connector to the workspace, when you configure a source using your connector. 
 
-The **connection** links up a configured source and a configured destination to perform syncs. It defines things like the replication frequency (e.g. hourly, daily, manually) and which streams to replicate.
+Follow [the tutorial](./tutorial.mdx) for an example of what this looks like in practice.
 
 ## Exporting the connector
 
 :::info
-This section is only relevant if you want to contribute your connector back to the Airbyte connector catalog to make it available outside of your workspace.
+If you choose to contribute your connector to the Airbyte connector catalog, making it publicly availavle outside of your workspace, you'll need to export it and go through the process of submitting it for review. 
 :::
 
-The connector builder leverages the [low-code CDK](https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/yaml-overview) under the hood, turning all configurations into the YAML format. While in most cases it's not necessary to interact with the YAML representation, it can be used to export the connector specification into a file and build a docker image containing the connector which can be shared more widely:
+The connector builder leverages the [low-code CDK](https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/yaml-overview) under the hood, turning all configurations into the YAML format. Typically, it's not necessary to interact with the YAML representation. However, you can export the connector specification into a file and build a docker image containing the connector which can be shared more widely:
```

Comment:
```suggestion
The connector builder leverages the [low-code CDK](https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/yaml-overview) under the hood, turning all configurations into the YAML format. Typically, it's not necessary to interact with the YAML representation. However, you can export the connector YAML into a file and build a docker image containing the connector which can be shared more widely:
```

Consider using `YAML` instead of `specification` since this matches the first sentence